anker/src/processor.rs

```diff
@@ -115,11 +116,6 @@ fn process_initialize(program_id: &Pubkey, accounts_raw: &[AccountInfo]) -> Prog
         accounts.ust_mint,
     )?;
 
-    let (_, token_swap_bump_seed) = Pubkey::find_program_address(
```

Comment:
Sneaked that here. In fact we don't really need this as it's stored in the token swap instance.